app/channel.js

```diff
@@ -10,7 +10,7 @@ const config = require('../js/constants/buildConfig')
 // The current channel is retrieved first from the buildConfig constants file,
 // then the environments and finally defaults to dev
 var channel = config.channel || process.env.CHANNEL || 'dev'
-let channels = new Set(['dev', 'beta', 'stable'])
+let channels = new Set(['nightly', 'developer', 'beta', 'dev'])
```

Comment:
can you add new strings below for nightly and developer in the `formattedChannel` method? 😄  Will require adding entries to https://github.com/brave/browser-laptop/blob/master/app/extensions/brave/locales/en-US/app.properties

Reply:
ef52180ed10eab223971c605854398adcaeea422